assets/src/edit-story/components/panels/captions/captions.js

```diff
@@ -82,18 +85,32 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
     [tracks, pushUpdate]
   );
 
-  const captionText = __('Upload captions', 'web-stories');
+  const handleChangeTrack = useCallback(
+    (attachment) => {
+      const newTracks = {
+        track: attachment?.url,
+        trackId: attachment?.id,
+        trackName: attachment?.filename,
+        id: uuidv4(),
+        kind: 'captions',
+        srclang: '',
+        label: '',
+      };
+
+      pushUpdate({ tracks: [...tracks, newTracks] }, true);
+    },
+    [tracks, pushUpdate]
+  );
 
   const UploadCaption = useMediaPicker({
     onSelect: handleChangeTrack,
     type: 'text/vtt',
     title: captionText,
-    buttonInsertText: __('Select caption', 'web-stories'),
+    buttonInsertText: __('Description', 'web-stories'),
```

Comment:
Why was this changed? It is wrong. 

![image](https://user-images.githubusercontent.com/237508/102091712-1e646d00-3e17-11eb-8c78-b8c5714154ae.png)

The button needs to be a call to action. Like `Select caption` or `Set captions`.

Reply:
Rollback change